### PR TITLE
Fix typos

### LINKED ISSUE #6246: Couple of possible typo's
In this section https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/sort-filter-page?view=aspnetcore-2.0#add-filtering-functionality-to-the-index-method says "Serve" and not "Server".
In this section: https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/sort-filter-page?view=aspnetcore-2.0#add-a-search-box-to-the-student-index-view 
Shouldn't the path be Pages/Students/Index.cshtml and not Views/Student/Index.cshtml because its Razor Pages?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 877f0457-37d9-9a34-e2ce-beb41d717086
* Version Independent ID: e9b0d4d1-e98c-fb96-415f-0ff4e37d3658
* Content: [Razor Pages with EF Core in ASP.NET Core - Sort, Filter, Paging - 3 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/sort-filter-page?view=aspnetcore-2.0#add-a-search-box-to-the-student-index-view)
* Content Source: [aspnetcore/data/ef-rp/sort-filter-page.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/sort-filter-page.md)
* Service: **unspecified**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #6246 

Thanks to @Pangar for reporting! :rocket:

### CODE REVIEW

File: aspnetcore/data/ef-rp/sort-filter-page.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ There's a performance penalty for calling `ToUpper`. The `ToUpper` code adds a f
 
 ### Add a Search Box to the Student Index View
```
Comment (scottaddie) [inline aspnetcore/data/ef-rp/sort-filter-page.md]: "Student Index View" doesn't make sense, since this isn't an MVC view.

**Comment by guardrex** [comment]
@scottaddie This might need even more work than this ... "view model" is present. Should I open a new issue to take a closer look at how this is handled later?

**Comment by Rick-Anderson** [comment]
@scottaddie  yes, open a new issue on that.

**Comment by guardrex** [comment]
https://github.com/aspnet/Docs/issues/6252

@Rick-Anderson I'll let you assign the labels.

**Pull request merged**